### PR TITLE
Create WG Template

### PR DESCRIPTION
**Pull request by jmertic**
For review by the TAC

### CODE REVIEW

**Comment by dheckenberg** [comment]
Thanks @jmertic.  We'll review at TAC meeting before merging.

**Comment by jmertic** [comment]
Thanks @dheckenberg ! Unless you think it should be reviewed by the TAC specifically, I'd say bring your suggestions into this PR and let's incorporate now.

File: process/wg_readme_template.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+_This document is intended to be a template for working groups established by the TAC, to be used as the README.md file in it's repository. All items in italics are descriptions of what context should be there; it is intended to be replaced by content. It is modeled off the [CNCF Serverless WG](https://github.com/cncf/wg-serverless)._
```
Comment (jfpanisset) [inline process/wg_readme_template.md]: it's repository should be its repository

File: process/wg_readme_template.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+_The intention is that each WG has it's own GitHub repo under the [ASWF organization](https://github.com/AcademySoftwareFoundation)._
```
Comment (jfpanisset) [inline process/wg_readme_template.md]: it's own GitHub repo should be its own GitHub repo

File: process/wg_readme_template.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+# _Working group name_
+
+_Describe the purpose of the group in no more that 4-5 sentences._
```
Comment (jfpanisset) [inline process/wg_readme_template.md]: Should there be a sentence as to what a "non-goal" is? I'm unclear as to whether this means:
* items that are in scope for discussion / WG activities but not considered a success metric
or
* items that are off the table and not to be considered by the WG

File: process/wg_readme_template.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+1. _goal 1_
+2. ...
+
+Non-goals of the WG are:
+
+1. _non goal 1_
+2. ...
+
+The TAC sponsor of this working group is _TAC sponsor name_
```
Comment (jfpanisset) [inline process/wg_readme_template.md]: maybe "The TAC member sponsor" to make it clear that WGs need to be sponsored by a specific member?

File: process/wg_readme_template.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+## Deliverables
+
+_provide a list of planned deliverables, or links to the deliverables. A best practice is to have directory named [deliverables](deliverables) in the repo for hosting and managing deliverables._
```
Comment (jfpanisset) [inline process/wg_readme_template.md]: is to have a directory

**Comment by dheckenberg** [comment]
@jmertic I agree with JF's suggestions.  Can you make these changes or would you like JF to submit them in subsequent PR?

**Comment by jmertic** [comment]
@dheckenberg I think since this PR is still open, it would be best for @jfpanisset to submit any changes to this one if that's ok.

**Comment by dheckenberg** [comment]
@jfpanisset could you submit your suggestions as changes to this PR?

**Comment by jfpanisset** [comment]
I'm not sure I know how to submit changes to an in-flight PR. I tried following the instructions at:

https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork

but was unable to push my changes back to the wg-template branch, presumably since I don't meet the pre-requisites for this workflow. Might be easier to just let this PR go in, and I'll submit these changes on top of it?

I really need to get better at git / github...

**Comment by jmertic** [comment]
Yeah - not sure what the issue is but I attempted to merge in your suggestions @jfpanisset. Let me know if I missed anything.

**Comment by dheckenberg** [comment]
@jfpanisset  Ok to merge?

**Comment by jfpanisset** [comment]
Yes, looks like my comments were incorporated.

**Pull request merged**